integrations/standalone/tests/integration/parts/start.ts

```diff
@@ -1,54 +1,72 @@
+import { TextArea } from '../../pageobjects/TextArea';
 
-export class StartTester implements PartTest {
-  constructor(private readonly hideParamDesc: boolean = false) {}
+class Start extends PartObject {
+  signature: TextArea;
+  paramSection: Section;
+  params: Table;
+  mapping: Table;
+  code: ScriptArea;
 
-  partName() {
-    return 'Start';
+  constructor(part: Part, private readonly hideParamDesc: boolean = false) {
+    super(part);
+    this.signature = part.textArea('Signature');
+    this.paramSection = part.section('Input parameters');
+    if (this.hideParamDesc) {
+      this.params = this.paramSection.table(['text', 'text']);
+    } else {
+      this.params = this.paramSection.table(['text', 'text', 'text']);
+    }
+    this.mapping = part.table(['text', 'label', 'expression']);
+    this.code = part.scriptArea();
   }
-  async fill({ page }: Part) {
-    await page.getByLabel('Signature').fill('myStart');
-    await CollapseUtil.open(page, 'Input parameters');
-    await TableUtil.addRow(page);
+
+  async fill() {
+    await this.signature.fill('myStart');
+    await this.paramSection.toggle();
+    const paramRow = await this.params.addRow();
     if (this.hideParamDesc) {
-      await TableUtil.fillRow(page, 0, ['param', 'String']);
+      await paramRow.fill(['param', 'String']);
     } else {
-      await TableUtil.fillRow(page, 0, ['param', 'String', 'desc']);
+      await paramRow.fill(['param', 'String', 'desc']);
     }
-    await TableUtil.fillExpression(page, 1, '"bla"');
-    await CodeEditorUtil.fill(page, 'ivy.log.info("hi");');
+    await this.mapping.row(1).column(2).fill('"bla"');
+    await this.code.fill('code');
   }
-  async assertFill({ page }: Part) {
-    await expect(page.getByLabel('Signature')).toHaveValue('myStart');
-    await CollapseUtil.open(page, 'Input parameters');
+
+  async assertFill() {
+    await this.signature.expectValue('myStart');
+    await this.paramSection.toggle();
+    const paramRow = this.params.row(0);
     if (this.hideParamDesc) {
-      await TableUtil.assertRow(page, 0, ['param', 'String']);
+      await paramRow.expectValues(['param', 'String']);
     } else {
-      await TableUtil.assertRow(page, 0, ['param', 'String', 'desc']);
+      await paramRow.expectValues(['param', 'String', 'desc']);
     }
-    await TableUtil.assertRow(page, 1, ['"bla"']);
-    await CodeEditorUtil.assertValue(page, 'ivy.log.info("hi");');
+    await this.mapping.row(1).column(2).expectValue('"bla"');
+    await this.code.expectValue('code');
   }
-  async clear({ page }: Part) {
-    await page.getByLabel('Signature').clear();
-    await CollapseUtil.open(page, 'Input parameters');
-    await TableUtil.removeRow(page, 0);
-    await TableUtil.fillExpression(page, 0, '');
-    await CodeEditorUtil.focus(page);
-    await CodeEditorUtil.clear(page);
+
+  async clear() {
+    await this.signature.clear();
+    await this.paramSection.toggle();
```

Comment:
Do you need this line?

Reply:
the param section / collapsable is per default closed. Was a wish by bb...